tests/test_funcs.py

```diff
@@ -448,6 +450,75 @@ def test_xp(self, xp: ModuleType):
         )
 
 
+@pytest.mark.skip_xp_backend(
+    Backend.SPARSE, reason="read-only backend without .at support"
+)
```

Comment:
```suggestion
```
this should now work.

Reply:
It doesn't have arange, but I updated the `reason`.